misk/src/main/kotlin/misk/security/authz/AccessInterceptor.kt

```diff
@@ -101,5 +90,16 @@ class AccessInterceptor private constructor(
 
   companion object {
     val logger = getLogger<AccessInterceptor>()
+
+    internal fun isAllowed(caller: MiskCaller, allowedCapabilities: Set<String>, allowedServices: Set<String>): Boolean {
```

Comment:
how do you feel about this being a method on MiskCaller?

Reply:
I think that'd make sense! 👍